tests/integration/multi_cluster_deploy_test.go

```diff
@@ -97,20 +97,6 @@ func (mrc *MultiRookClusterDeploySuite) TearDownSuite() {
 	mrc.installer.CleanupCluster(clusterNamespace1)
 	mrc.installer.CleanupCluster(clusterNamespace2)
 
-	//Delete ClusterRole and ClusterRoleBindings
```

Comment:
this moved to inside the CleanupCluster function just above because that function actually deletes the namespace.  We need to clean up roles/bindings before deleting the namespace they are in.